backends/dpdk/DpdkXfail.cmake

```diff
@@ -25,6 +25,7 @@ p4c_add_xfail_reason("dpdk"
   testdata/p4_16_samples/pna-dpdk-wrong-warning.p4
   testdata/p4_16_samples/pna-dpdk-invalid-hdr-warnings5.p4
   testdata/p4_16_samples/pna-dpdk-invalid-hdr-warnings6.p4
+  testdata/p4_16_samples/pna-dpdk-header-union-stack2.p4
```

Comment:
This hits a bug in the header copy elimination pass in dpdk backend when lhs and rhs of assignment are both local variables. The pass expects that local to local variable copy should be eliminated by local copy propagation pass. This will need further analysis as to where it should be fixed.